code/modules/power/groundmap_geothermal.dm

```diff
@@ -309,6 +312,11 @@
 	name = "\improper Reinforced Reactor Turbine"
 	is_corruptible = FALSE
 
+///This one spawns intact
+/obj/machinery/power/geothermal/intact
+	corrupted = 0
+	buildstate = GEOTHERMAL_NO_DAMAGE
```

Comment:
???

Reply:
Added for the sake of more tools in our toolbox.